parking_permits/services/hel_profile.py

```diff
@@ -34,6 +34,9 @@ def get_customer(self):
             "last_name": self.__profile.get("lastName"),
             "email": email_node.get("email") if email_node else None,
             "phone_number": phone_node.get("phone") if phone_node else None,
+            "national_id_number": self.__profile.get("verifiedPersonalInformation").get(
```

Comment:
Minor, but if I'm not mistaken, the `verifiedPersonalInformation` could be `None`. So we probably want to do something like:

```suggestion
            "national_id_number": self.__profile.get("verifiedPersonalInformation", {}).get(
```

Reply:
Done.